server/build_event_protocol/build_event_server/build_event_server.go

```diff
@@ -37,12 +43,18 @@ func NewBuildEventProtocolServer(env environment.Env) (*BuildEventProtocolServer
 }
 
 func (s *BuildEventProtocolServer) PublishLifecycleEvent(ctx context.Context, req *pepb.PublishLifecycleEventRequest) (*emptypb.Empty, error) {
+	wg := &sync.WaitGroup{}
```

Comment:
nit: I would suggest using `errgroup` instead. It uses a waitgroup underneath but has a slightly more ergonomic interface . You can return `nil` error if you don't care about errors.

Reply:
Done